frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -620,23 +620,25 @@ export class TileViewPage extends React.Component {
   }
 
   renderSidePanel() {
-    let { renderFilterGroup } = this.props;
+    let { isOperatorHub, renderFilterGroup } = this.props;
     const { activeFilters, filterCounts } = this.state;
 
     renderFilterGroup = renderFilterGroup || this.renderFilterGroup;
 
     return (
       <FilterSidePanel>
-        <FilterSidePanel.Category key="keyword" onSubmit={(e) => e.preventDefault()}>
-          <FormControl
-            type="text"
-            inputRef={(ref) => this.filterByKeywordInput = ref}
-            placeholder="Filter by keyword..."
-            bsClass="form-control"
-            value={activeFilters.keyword.value}
-            onChange={e => this.onKeywordChange(e.target.value)}
-          />
-        </FilterSidePanel.Category>
+        {!isOperatorHub &&
```

Comment:
We want the filter in the same spot for OperatorHub and Developer Catalog. I don't think we need to special case anything

Reply:
Great, I'll make that change.